presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/CommandExecutor.java

```diff
@@ -112,10 +107,12 @@ private String getProcessOutput(Process process)
         catch (InterruptedException e) {
             LOGGER.error(e, "Process failed to complete.");
         }
+        // TODO: Should this throw an exception instead of returning null?
+        // How will the caller handle this null?
```

Comment:
@ShanBai6 Control reaches this point after an exception is caught, because the `catch`-blocks don't exit the function.